app/javascript/src/constants/formValidationSchemas.js

```diff
@@ -26,5 +26,7 @@ export default {
   notesForm: Yup.object().shape({
     title: Yup.string().required("Title is required"),
     description: Yup.string().required("Description is required"),
+    contact: Yup.object().required("Contact is required"),
+    tags: Yup.array().required("Tag is required").nullable(true),
   }),
 };
```

Comment:
Please look into my suggestion for moving this to a constants file in the scope of this directory.